angular-ngrx-scss/src/app/shared/components/issue-pull-item/issue-pull-item.component.html

```diff
@@ -0,0 +1,48 @@
+<div *ngFor="let item of itemsList">
+  <div class="item">
+    <div class="item-container">
+      <div class="info">
+        <ng-container *ngIf="item.state === 'open'; else closed">
```

Comment:
Probably not necessary, but might be nice to have the states in an enum and use a mapping method in the template.

Reply:
probably this will be addressed when we connect the Github API with the view, but good point @honzikec